sources/packages/api/src/route-controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -248,37 +248,23 @@ export class ProgramInfoRequestController extends BaseController {
         throw new BadRequestException("Application not found.");
       }
 
-      let studyStartDate = payload.studyStartDate;
-      let studyEndDate = payload.studyEndDate;
-      let selectedOfferingIntensity = payload.offeringIntensity;
-
-      let offeringToCompletePIR: EducationProgramOffering;
-      if (payload.selectedOffering) {
-        // Check if the offering belongs to the location.
-        const offeringLocation =
-          await this.offeringService.getOfferingLocationId(
-            payload.selectedOffering,
-          );
+      // Check if the offering belongs to the location.
+      const offeringLocation = await this.offeringService.getOfferingLocationId(
+        payload.selectedOffering,
```

Comment:
since we are doing  `dryRunSubmission` here, maybe instead of `payload` you can also use `submissionResult.data.data`

Reply:
The payload now is supposed to be only the `offeringId` that is already validated using class decorators to be a positive value. 
@henryoforeh-aot, I do not see a reason to keep doing the `dryRun` for this method. It was in place before because of the possible offering creationg.